CommonLibSF/include/SFSE/API.h

```diff
@@ -19,4 +19,29 @@ namespace SFSE
 
 	Trampoline& GetTrampoline();
 	void        AllocTrampoline(std::size_t a_size, bool a_trySFSEReserve = true);
+
+	namespace stl
+	{
+		template <typename T>
+		constexpr void write_thunk_call() noexcept
```

Comment:
- CLIB has consistently used `<class T>` etc when defining templates
- add `write_call<6>` variants, perhaps rename the existing one to `write_thunk_call_5` or similar
- Capitalizing `Thunk` -_-

Reply:
I only capitalized it bc function names are CamelCase in general in CLib 😅 

Will change to `<class T>` and add a `write_call<6>` variant